llm/generate/gen_windows.ps1

```diff
@@ -1,6 +1,6 @@
 #!powershell
 
-$ErrorActionPreference = "Stop"
+# $ErrorActionPreference = "Stop"
```

Comment:
Did you mean to comment this out?

Reply:
I'll remove the line.  Start-ThreadJob has some quirks around how errors are interpreted (the biggie is any stderr output is considered and error, ugh). exit also doesn't work in the thread job, so I had to switch to throw to get error handling to work right and detect a failed job.